apps/whale-api/custom.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module 'browserify-aes'
+declare module 'bip66'
```

Comment:
These came up complaining and has no typing available so added module declares here.

Reply:
https://github.com/DeFiCh/jellyfish/pull/1027